langcodes/db.py

```diff
@@ -152,7 +152,12 @@ def add_language(self, data, datalang, name_order=0):
             'language',
             (subtag, script, is_macro, is_collection, preferred, macrolang)
         )
-        if not 'Preferred-Value' in data:
+        if 'Preferred-Value' in data:
+            desc = ';'.join(data.get('Description', []))
```

Comment:
The 'Description' field is a list because that's how it's parsed in registry_parser.py, and coping with multiple values by joining them with semicolons is the same thing we do with other 'Description' fields in db.py. But the 'Descriptions' of language mappings are never actually used for anything, so there's not really anything to test. Maybe that means they should be removed in some version.

Now, looking at this made me realize that the thing the Description is used for -- setting a default English name for a language -- was being skipped for languages that have been merged into another language. We should still recognize the old name, so I took out the `else:` and added a case for that.